app/src/signing/networkMessages/cosmos-transactions.js

```diff
@@ -109,14 +106,12 @@ export async function getSignableObject(
 
 export async function getBroadcastableObject(
   chainMessages,
-  { accountSequence, accountNumber, gasEstimate, fee, memo },
```

Comment:
:eyes:

Reply:
those don't need to be in and break cosmos-hub-4 txs. I tested on gaia and it is fine

Reply:
:ok_hand: